### PR TITLE
Homer scan motif genome wide + small fix

### PR DESCRIPTION
**Pull request by lldelisle**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Hi,
- I corrected a bug in homer_findMotifsGenome when using homer1.
- I decided to add as new datasets the motifs found by this tool. I decided to use the existing txt datatype. It would be probably better to create a datatype 'homer_motif' (a description of the motif file is [here](http://homer.ucsd.edu/homer/motif/) in section 12).
- I added a new tool to the collection which is `scanMotifGenomeWide`.

### CODE REVIEW

File: tools/homer/homer_findMotifsGenome.xml [hunk 1]
```diff
@@ -115,20 +115,24 @@ $advanced.oligo
...
     -minlp '$advanced.homer12.minlp'
 #elif str( $advanced.homer12.version ) == "homer1":
+    -homer1
     -depth '$advanced.homer12.depth'
 #end if
 #if str( $background.use ) == "none":
     #if not $nomotif:
         && cp '${output}'/homerResults.html outputHomer.html
         && cp -r '${output}' '${html_homer_file.files_path}'
+        && cp -r '${output}'/homerResults homerResults
```
Comment (bgruening) [inline tools/homer/homer_findMotifsGenome.xml:125]:
```suggestion
        && cp -r '${output}/homerResults' homerResults
```

Comment (bgruening) [reply tools/homer/homer_findMotifsGenome.xml:125]: and below

File: tools/homer/homer_scanMotifGenomeWide.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        </conditional>
+        <param argument="-bed" type="boolean" truevalue="-bed" falsevalue="" checked="false" label="Output bed format."/>
+        <param argument="-keepAll" type="boolean" truevalue="-keepAll" falsevalue="" checked="false" label="Report all motifs even if overlap"
+        help="By default, HOMER will remove a motif that overlaps itself, useful for palindromes. To report all, specify this option."/>
+        <param argument="-mask" type="boolean" truevalue="-mask" falsevalue="" checked="false"
+        label="Do not look for motifs in RepeatMasked sequence (lower case sequence in FASTA files)"/>
+        <param argument="-5p" name="Fivep" type="boolean" truevalue="-5p" falsevalue="" checked="false"
+        label="Report motif positions based on the 5' end of the motif sequence."/>
+        <param argument="-homer1" type="boolean" truevalue="-homer1" falsevalue="" checked="false"
+        label="Use homer1 instead of homer2."/>
```
Comment (bgruening) [inline tools/homer/homer_scanMotifGenomeWide.xml:55]: I would remove the trailing dot

File: tools/homer/homer_scanMotifGenomeWide.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    <help><![CDATA[
+
+.. class:: infomark
+
+    This is a wrapper for scanMotifGenomeWide.pl from HOMER but not all options are included.
+
+    The scanMotifGenomeWide.pl script will take a motif file (may contain multiple motifs) and look for instances across the genome. 
+
+Usage::
+    scanMotifGenomeWide.pl <motif> <genome> [-5p] [-homer1/2] [-bed] [-keepAll] [-mask]
```
Comment (bgruening) [inline tools/homer/homer_scanMotifGenomeWide.xml:93]: can/should we remove the commandline here?

**Review by bgruening — COMMENTED** [review]
Thanks.

**Comment by lldelisle** [comment]
Thanks for the review.
CI is failing because I put a fasta of 2MB.
I need to reduce its size.

**Comment by bgruening** [comment]
@lldelisle don't worry about the failed size tests. Old tools are fine to fail.

**Comment by lldelisle** [comment]
Too late... Let see if the tests pass like this...

**Comment by lldelisle** [comment]
It passed...

**Comment by lldelisle** [comment]
Ready for review.

**Review by bgruening — APPROVED** [review]
Thanks @lldelisle

**Comment by lldelisle** [comment]
Thanks

**Pull request merged**